components/Divider.tsx

```diff
@@ -0,0 +1,6 @@
+import { Separator } from "@shadcn/ui/separator";
+
+
```

Comment:
**praise:** We should have Prettier installed to VS Code to not have two empty lines.